testsuite/classlibrary/TestLcmGcd.sc

```diff
@@ -0,0 +1,156 @@
+
+TestLcmGcd : UnitTest {
+
+
+	// lattice theoretic laws
+	// https://en.wikipedia.org/wiki/Least_common_multiple#Lattice-theoretic
+
+	callTest_commutative_lcm { |a, b|
+		var x = lcm(a, b);
+		var y = lcm(b, a);
+		this.assertEquals(x, y, "lcm(%, %) = lcm(%, %) should be valid"
```

Comment:
yes, this is true. How would you do the test so that when some of them fail you know which ones? I ask specifically because brian convincingly argued that each test should have a smaller scope, so that it is easier to analyze.